app/gui/view/src/project/project_view_top_bar/window_control_buttons.rs

```diff
@@ -0,0 +1,199 @@
+//! The component with buttons in the top left corner. See [[View]].
```

Comment:
The file is named "window_control_buttons", while the module docs tell that this is a component with buttons in the top left corner. So is this module defining window control buttons or just a component with buttons in some corner?

Reply:
We are re-evaluating moved code in other PRs. If the docs does not make sense, they need to be fixed.

Reply:
> We are re-evaluating moved code in other PRs

Do we? If yes, then I'm against it as a rule, as this make a delivery of tasks very unpredictable. This very PR was meant to implement a 1-day task with high priority, we definitely should not reevaluate older code.

Reply:
I have pretty often comments in my PR from different people, uncluding you Adam, about old code that was moved that is not up to date anymore. Some things, like this thing, where the docs are just unclear and fixing them is 1-minute job should be fixed if the reviewer catches them - this is also what we've been doing in many PRs in the past. 

Nevertheless, @Procrat, please fix these docs in the next PR, as this PR was already merged.